src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,52 @@
+.calendar {
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  margin: 0 auto;
+  width: ($day-size * 7) + ($gap-size * 6) + ($calendar-padding * 2);
+
+  &__container {
+    display: flex;
+    flex-wrap: wrap;
+    gap: $gap-size;
+    padding: $calendar-padding;
+
+    @each $day, $number in $week {
+      &--start-day--#{$day} > :first-child {
+        margin-left: ($day-size + $gap-size) * $number;
+      }
+    }
+
+    @for $last-day from 29 through 31 {
```

Comment:
```suggestion
    @for $last-day from 28 through 31 {
```

Reply:
maybe I wouldn't notice it on another day but today is the 28 of Feb and it is the last day of the month)